core/unix/os_exports.h

```diff
@@ -295,7 +312,14 @@ disable_env(const char *name);
 #        define DECLARE_DATA_SECTION(name, wx)     \
             asm(".section " name ", \"a" wx "\""); \
             asm(".align 12"); /* 2^12 */
-#    endif                    /* X86/ARM */
+#    elif defined(DR_HOST_RISCV64)
+/* FIXME-RISCV: RFC: Why all archs align those sections to 4K? Page size?
```

Comment:
DR does change page protections for different data regions at runtime -- looking at this I wonder if that's broken for >4K pages.  It is only on by default on Windows maybe IIRC.

Reply:
Yes, DR has page size computation and dynamic page size handling everywhere else: but it looks like this is a case where it does not do the right thing; but like I said AFAIK it is not enabled anywhere where it has been hit yet.  I filed #5565 on this.  Please include it in this comment; probably can be downgraded to XXX instead of FIXME.

Reply:
Done.